package.json

```diff
@@ -17,5 +17,8 @@
     "pnpm": "^10.13.1",
     "turbo": "^2.5.5",
     "typescript": "^5.8.3"
+  },
+  "dependencies": {
```

Comment:
update vitest to be just a devDependency. Use pnpm uninstall vitest and then pnpm add -D vitest